modules/help_editor/js/help_editor_helper.js

```diff
@@ -35,7 +34,51 @@ $("input[name=preview]").click(function(e) {
      e.preventDefault();
 
 });
+$("#save-help").click(function(e) {
+    e.preventDefault();
+    var title   = $('input[name="title"]').val(),
+        content = $('textarea[name="content"]').val(),
+        section = $("#section").val(),
+        subsection = $("#subsection").val(),
+        parentID = $("#parentID").val(),
+        helpID = $("#helpID").val(),
+        returnString = $("#return").val();
+
+    $.ajax({
+        type: 'POST',
+        url: loris.BaseURL + '/help_editor/ajax/process.php',
+        data: {
+            title: title ? title : '',
+            content: content ? content : '',
+            section: section ? section : '',
+            subsection: subsection ? subsection : '',
+            parentID: parentID ? parentID : '',
+            helpID: helpID ? helpID : '',
```

Comment:
not a blocker, but I think these would read better as `title: title || ''` (etc)